conda-store-server/conda_store_server/server/views/ui.py

```diff
@@ -159,6 +161,20 @@ def ui_get_build(build_id):
     return render_template("build.html", **context)
 
 
+@app_ui.route("/user/", methods=["GET"])
+def ui_get_user():
```

Comment:
Ahh. I think you are thinking of this differently. Github has a route that should be called within the callback_url method https://api.github.com/user to get the username and organizations that the user is a member of.

Reply:
And conda-store shouldn't keep any information about the user. I'm saying this because every authentication provider is a bit different and may not provide an email/username etc. Additionally not everything logged into conda-store is a user. Often times it will be a service that has authenticated via a conda-store token. So conda-store should only be concerned about storing the authorization that the given authenticated user has (unless we want to handle to complexity of is this a user or a service etc. which I think is unneeded).

Reply:
I am doing that [here](https://github.com/Quansight/conda-store/pull/103/files#diff-8aca98d789e72a0f566677c4fd621c36710f5f7962296ae7ba9775a69bb3bebfR504) and then saving the user data as json cookie. I didn't get the organizations yet to save a query and because I want to know how generic this needs to be (organizations might be GH-specific).